### PR TITLE
Liberation Station: Babby's first pull request

### PR DESCRIPTION
**Pull request by MMMiracles**
image

Adds a new vending machine called the Liberation Station, equipped with the finest ballistic weapons a badmin could ever need for gimmick events.

Hack the machine and get your very own Evel Kn- I mean patriotism suit, comes with its very own mini-cape! 

Now includes patriot bedsheets so you can sleep in the silky softness that is freedom itself.

### CODE REVIEW

**Comment by SconesC** [comment]
Oh SHIT

Can you make a list of things that come in it? I hope it has an American flag bedsheet/cloak!

**Comment by pudl** [comment]
oh my god

**Comment by MMMiracles** [comment]
@SconesC oh fuck why didn't I think of adding the american flag bedsheet brb.

**Comment by SconesC** [comment]
So now we can Evel Kn- Patriotism Suits
High-caliber 'self defense' weapons for liberating our homes/workplaces
And American flags? 

It should dispense burgers as well

**Comment by Incoming5643** [comment]
It should accept burgers into its coin slot, and people should be forced to clap emote every time they select something

**Comment by MMMiracles** [comment]
pssh i dont know to get rid of it hnng

**Comment by ike709** [comment]
As a patriot, I approve. But I am waiting for one of the tea drinking brits to tell you to add a Union Jack.

File: tgstation.dme [hunk 1]
```diff
@@ -1,1407 +0,0 @@
-// DM Environment file for tgstation.dme.
```
Comment (Iamgoofball) [inline tgstation.dme]: You broke it

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -927,6 +927,21 @@
 	contraband = list(/obj/item/weapon/stock_parts/cell/potato = 3)
 	premium = list(/obj/item/weapon/storage/belt/utility = 3)
 
+/obj/machinery/vending/liberationstation
+	name = "\improper Liberation Station"
+	desc = "An overwhelming amount of ancient patriotism washes over you just by looking at the machine."
```
Comment (Iamgoofball) [inline code/game/machinery/vending.dm]: "Ancient patriotism" should be bold and span class userdanger to get that extra freedom into the description

**Comment by Chocobro** [comment]
this is fucking funny

**Comment by xxalpha** [comment]
muh immersions though

**Comment by WJohn** [comment]
Oh god why do you people still play with stretch to fit.

I'd give it :+1: but right now the pictures are so hideously deformed I can't discern the shading and highlights at all.

**Comment by MMMiracles** [comment]
Updated with non-stretched pictures.

**Comment by MMMiracles** [comment]
alright i can't even at this point. someone with more experience please come to my rescue.

**Comment by Incoming5643** [comment]
Go here and merge this and hopefully everything will work out fine https://github.com/MMMiracles/-tg-station/pull/1

**Comment by MMMiracles** [comment]
if this works then you are absolutely based

**Comment by Incoming5643** [comment]
Your line count is fucked but that part of this seems to have been fixed. Something else is still wrong though.

**Comment by Incoming5643** [comment]
Travis retest this

**Comment by RemieRichards** [comment]
Wake up travis ya dingus.

**Comment by Incoming5643** [comment]
Travis is kill

**Comment by MMMiracles** [comment]
oy vey

**Comment by Cheridan** [comment]
You'll need to use the map merger.

**Comment by xxalpha** [comment]
He's not editing maps.

**Comment by RemieRichards** [comment]
Cheridan cray cray.

**Comment by Cheridan** [comment]
oh

I saw  tgstation.dme +1400 -1400 and thought it was a map

**Comment by MMMiracles** [comment]
so is there not much i can do at this point im guessing?

**Comment by AnturK** [comment]
Do soft reset and commit again without the dme

**Comment by Incoming5643** [comment]
yeah just set aside the files you meant to change somewhere outside the git folder, go into a new branch, paste them over the unedited version, and commit that without the dme.

You never commit the dme unless you've made a whole new file during your changes

**Comment by MMMiracles** [comment]
alright round 2 lets go github

**Comment by pudl** [comment]
FREEDOM IS KILL

**Pull request closed without merging**